qt/aqt/main.py

```diff
@@ -1431,7 +1431,8 @@ def show_menubar(self) -> None:
     def setup_auto_update(self, _log: list[DownloadLogEntry]) -> None:
         from aqt.update import check_for_update
 
-        check_for_update()
+        if aqt.mw.pm.enable_update_check():
```

Comment:
`update_check_enabled` or `check_for_updates` for the function / key name would be a bit clearer/more consistent